src/planet-4-151612/wordpress/etc/my_init.d/20_install_wordpress.sh

```diff
@@ -279,8 +279,7 @@ _good "Database ready: ${WP_DB_HOST}:${WP_DB_PORT}"
 # FIXME If installed, perform site-update?
 
 wp core install --url="${WP_HOSTNAME}" --title="$WP_TITLE" --admin_user="${WP_ADMIN_USER:-admin}" --admin_email="${WP_ADMIN_EMAIL:-$MAINTAINER_EMAIL}"
-
-wp plugin activate --all
+wp plugin activate --all --skip-plugins=wordfence
```

Comment:
This unfortunately doesn't work to exclude the `wordfence` plugin, we probably have to call `wp plugin list` and pass the results to `wp plugin activate` after manually excluding Wordfence.

Reply:
I wonder why we activate all plugins here :thinking: When it releases and connects to the DB it will use the actual activated plugins on the site. And if I'm not mistaken it runs these init scripts again anyway.